shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -1866,4 +1866,31 @@ - (void)testFlutterTextInputPluginHostViewNotNil {
   XCTAssertNotNil([flutterEngine.textInputPlugin hostView]);
 }
 
+- (void)testSetPlatformViewClient {
+  FlutterViewController* flutterViewController = [FlutterViewController new];
```

Comment:
Do not use +new: https://google.github.io/styleguide/objcguide.html#do-not-use-new

Seems other tests in this file also use +new, we should fix those later (low priority)